documentation/docs/student/assessment.mdx

```diff
@@ -0,0 +1,110 @@
+- **Automatic:** Your submissions are automatically assessed (e.g., quiz exercises, programming exercises with test cases).
+- **Semi-Automatic:** Automatic assessment provides a starting point, which reviewers then improve manually.
+
+After an exercise's due date passes, your submission will be assessed according to the configured assessment mode.
+Once the assessment is complete and the assessment due date has passed, you'll receive detailed feedback on your work.
+
+
+## Double-Blind Assessment
+
+Manual assessments in Artemis are **double-blind**:
+
+- Reviewers don't see your name when grading your submission
+- You don't see the reviewer's identity
+
+This increases objectivity and minimizes bias in the assessment process.
+
+
+## Receiving Feedback
+
+After the assessment due date, you can view your:
+
+- **Score:** Points achieved out of the total possible points
+- **Feedback:** Detailed comments on your submission, often organized by grading criteria
+- **Assessment:** Any inline annotations or corrections made by reviewers
+
+<Callout variant={CalloutVariant.info}>
+    <p>For programming exercises with automatic grading, you may receive immediate feedback during the working period if configured by your instructor.</p>
+</Callout>
+
+
+## Complaints
+
+If you believe an assessment was incorrect or unfair, you can submit a **complaint** _(if enabled by your instructor)_.
+Submitting a complaint is a request for a re-evaluation of your submission.
+
+### How Complaints Work
+
+1. **Complaint Tokens:** Your instructor sets a maximum number of complaints you can submit per course. Each complaint uses one token.
+2. **Token Return:** If your complaint is accepted, the token is returned to you. Throughout a course you can submit as many complaints as you want, as long as they are accepted.
+3. **Deadline:** Complaints must be submitted before the complaint due date set by your instructor.
+
+### Submitting a Complaint
+
+1. Open the assessed exercise
+2. Click the **"Complain"** button below the exercise instructions
+3. Write a clear justification explaining why the assessment should be reconsidered
+4. Submit your complaint
+
+<Image src={complaintSubmissionImg} alt={"Complaint Submission"} size={ImageSize.large}></Image>
+
+A reviewer will evaluate your complaint and either:
+- **Accept it:** Your score may be adjusted (up or down) with additional feedback
+- **Reject it:** The reviewer will explain why the original assessment stands
+
+<Callout variant={CalloutVariant.warning}>
+    <p>Once you submit a complaint, you cannot submit a "More Feedback Request" for the same exercise.</p>
+</Callout>
+
+
+## More Feedback Requests
+
+If you want additional clarification or feedback but don't want to dispute your score, you can submit a **More Feedback Request**.
+
+<Image src={moreFeedbackImg} alt={"More Feedback Request"} size={ImageSize.large}></Image>
+
+<Callout variant={CalloutVariant.danger}>
```

Comment:
I think the emoji for the danger callout should be changed to ❗ instead.

Having 🚫 makes it look like a ‘do not ever do that’ item. You still might want to do dangerous stuff but only after thinking carefully about it.
❗ is also used in other places for dangerous stuff (e.g. traffic warning sign).